automation/jleechanorg_pr_automation/check_codex_comment.py

```
@@ -5,17 +5,17 @@
 
 import json
 import sys
+from typing import Tuple
```

Ruff (0.14.8)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)